fundamentals/trees/test/trees.test.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+const fs = require('fs');
+const path = require('path');
+
+// jest and jsdom don't play nice with each other so we have to
+// inject the html and script into our mock as strings
+const htmlString = fs
+  .readFileSync(path.join(__dirname, '..', 'trees.html'))
+  .toString();
+
+const scriptString = fs
+  .readFileSync(path.join(__dirname, '..', 'trees.js'))
+  .toString();
+
+const domOptions = { runScripts: 'dangerously' };
+const dom = new JSDOM(htmlString, domOptions);
+
+const { document } = dom.window;
+
+const body = document.querySelector('body');
+const scriptEl = document.createElement('script');
+scriptEl.innerHTML = scriptString;
+body.appendChild(scriptEl);
```

Comment:
You probably could have made this a utility function you imported, so that students don't worry about it.